scripts/cfos/make_dataset_data_tables.py

```diff
@@ -87,113 +60,185 @@ def generate_load_table_files(dataset_tables_dict):
 
-        # TODO: edit the value = dataframe for a table to have the entity:table_name_id format
+    if has_err:
+        print(f"Errors found, see {VALIDATION_ERROR_FILE} for details.")
+        exit()
 
-        # add KVP {table name : table_df} entry to all tables dictionary
-        dataset_tables_dict[table] = table_df
+    print(f"Success: Dataset has been validated.")
+    return validated_dataset
 
-    print(f"Success: Dataset has been partitioned into {len(dataset_table_names)} tables: {dataset_table_names}")
-    return dataset_tables_dict
 
+def col_is_in_defined_fields(col_name, field_dict):
+    list_of_fields = list(field_dict.keys())
+    if col_name in list_of_fields:
+        return True
 
-def validate_dataset(dataset):
-    """Validate the full dataset metadata with Schema validator."""
+    return False
 
-    # capture errors from the validation against the pre-defined validations per column
-    errors = schema_validator.validate(dataset, columns=schema_validator.get_column_names())
-    # get indices of the failed rows
-    errors_index_rows = [e.row for e in errors]
 
-    # if any errors, print error message and exit
-    if errors_index_rows:
-        print(f"Failed: Dataset validation errors found. Please retry after correcting errors listed in validation_errors.csv.")
-        pd.DataFrame({'validation_error':errors}).to_csv('validation_errors.csv')
-        exit()
+def load_excel_input(excel, allowed_dataset_cols, allowed_dataset_tables, skiprows):
+    """
+    Read excel file input into a dataframe and validate for required columns.
+        Skipping first two rows, as the input excel has two rows of headers before the data starts.
+        Usecols uses a lambda function to filter out columns that aren't defined in the schema.json file's 
+            fields definition section. This is used instead of a list of expected columns, because providing
+            a list enforces that exactly those columns must exist in the data frame, and we don't want to
+            enforce the schema until we're checking it during validation. This should solely be data import.
 
-    # TODO: determine how to handle drops of failed rows/columns to get cleaned data
-    # errors_index_rows = [e.row for e in errors]
-    # data_clean = dataset_metadata_df.drop(index=errors_index_rows, axis=1)
-    # data_clean.to_csv('clean_data.csv')
-    print(f"Success: Dataset has been validated.")
-    return dataset
+        Returns: dictionary of data frames where each sheet is a separate dataframe in the dictionary
+    """
 
+    raw_dataset_dict = pd.read_excel(excel, sheet_name=None, skiprows=skiprows, index_col=None, usecols=lambda x: x in allowed_dataset_cols)
 
-def load_excel_input(excel, expected_dataset_cols):
-    """Read excel file input into a dataframe and validate for required columns."""
+    print("Success: Excel file has been loaded into a dataframe. Note that any columns that are not defined in the schema will be ignored.")
```

Comment:
UX suggestion: tell the user how many rows and columns of data were read in, as a sanity check.

Reply:
I'm not sure this is worth doing. I'd have to loop over the dictionary for each data frame and print the name of the table and the size of the data, and even then, they won't be able to tell what's missing if something is missing. 

Would it make more sense to just print the dictionary of data frames?